content/en/serverless/installation/nodejs.md

```diff
@@ -261,9 +259,11 @@ Replace `<TAG>` with either a specific version number (for example, `7`) or with
 {{% /tab %}}
 {{% tab "Custom" %}}
 
-### Install
+<div class="alert alert-warning">If you are not using any of the recommended serverless development tools, such as the Serverless Framework, we strongly encourage you instrument your serverless applications with the <a href="/?tab=datadogcli">Datadog CLI</a>.</div>
```

Comment:
In both places, let's move this message before the tabs (line 37). That way, the reader sees it before they start working through the instructions, and no matter which tab is selected on the page. And I think if we do that, we can lower the class to alert-info.

Reply:
```suggestion
<div class="alert alert-warning">If you are not using a serverless development tool, such as the Serverless Framework or AWS CDK, instrument your serverless applications with the <a href="/?tab=datadogcli">Datadog CLI</a>.</div>
```
Under what circumstances would someone use Custom? Can we add that info?

Reply:
I actually prefer leaving the message on only when the custom tab is selected, because the customer is using one of the other supported installation method, say Serverless Framework, I'd rather not mislead/distract them.

I do think reducing the level to info makes sense, since it's really not a warning but a recommendation.

I cannot actually think of a good use case for custom installation, and I assume the customer would proceed and ignore my recommendation if the CLI doesn't work for them (need to learn why as a follow up action item).